miden/tests/integration/air/memory.rs

```diff
@@ -18,18 +19,30 @@ fn pushw() {
 
 #[test]
 fn pop() {
-    let asm_op = "pop.mem.0";
-    let pub_inputs = vec![1];
+    let asm_op = "pop.mem.0 pop.mem.0 pop.mem.0";
+    let pub_inputs = vec![1, 2];
 
-    build_op_test!(asm_op, &pub_inputs).prove_and_verify(pub_inputs, 0, false);
+    build_op_test!(asm_op, &pub_inputs).prove_and_verify(pub_inputs.clone(), 0, false);
+
+    let exe_trace = build_op_test!(asm_op, &pub_inputs).execute().unwrap();
+    let helper_regs1 = [2, 0, 0, 0, 0, 0].to_elements();
+    assert_eq!(helper_regs1, exe_trace.get_helper_state(5));
```

Comment:
Could you add a brief comment explaining why helper registers are set at cycle 5? Maybe just listing a sequence of operations which gets executed. E.g., the first operation is probably `SPAN`, then something else, then something else etc.